calico/felix/devices.py

```diff
@@ -54,6 +56,11 @@ def check_kernel_config():
                       "0 or 1.")
         raise BadKernelConfig("net.ipv4.conf.all.rp_filter set to 'loose'")
 
+    # Make sure the default for new interfaces is set to strict checking so
```

Comment:
Oops, one is the 'all', the other is 'default'.  I think all that's needed is to add a note that 2 in all overrides specific configuration to 1; which we set for all interfaces (specifically and as the default for new interfaces).